messaging/infinispan-grpc-kafka/README.md

```diff
@@ -1,7 +1,7 @@
 # Table of Contents
 1. [Quarkus infinispan grpc kafka](#Quarkus-infinispan-grpc-kafka)
-2. [Quarkus SSL/TLS Infinispan scenario ](#Quarkus-SSL/TLS-Infinispan-scenario )
-3. [Quarkus Grateful Shutdown for Kafka connectors](#Quarkus-Grateful-Shutdown-for-Kafka-connectors)
+2. [Kafka-client SASL](#Kafka-client-SASL)
```

Comment:
We have SASL SSL scenario in this module, please rename

Reply:
IMHO chapters in the readme does not need to to 1on1 match to the scenarios. Other sections are also not doing it, but OK I will change it.